sdk/src/dataMapper/SchemaDataMapper.ts

```diff
@@ -63,7 +63,7 @@ export default class SchemaDataMapper extends BaseDataMapper<Schema, Schema_filt
   }
 
   async getSchemasNumber() {
-    return this.executeReadMethod("getSchemasNumber", []);
+    return await super.findTotalCount();
```

Comment:
I don't think we need to keep this `await` here.
Actually, I think we can remove a lot of unnecessary `await` in this file!